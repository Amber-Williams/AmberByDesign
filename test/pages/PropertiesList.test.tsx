// import React from 'react'
// import { render } from '../testUtils'
// import CaseStudyListItem from '../../components/PropertiesList/CaseStudyListItem'

// describe('CaseStudyListItem', () => {
//   it('matches snapshot', () => {
//     const property = {
//       id: 1,
//       images: ['www.img1.com', 'www.img2.com', 'www.img3.com'],
//       numberBedrooms: 2,
//       address: '100 blah blah road',
//       postcode: 'SE1 3TL',
//       features: ['nice', 'big', 'pool'],
//       description: 'Blah blah balh',
//       price: 'Â£100,000',
//       expired: false,
//       dateAdded: '3-5-2020',
//     }

//     const { asFragment } = render(
//       <CaseStudyListItem property={property} />,
//       {}
//     )
//     expect(asFragment()).toMatchSnapshot()
//   })
// })
