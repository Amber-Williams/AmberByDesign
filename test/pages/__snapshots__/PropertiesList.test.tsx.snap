// Jest Snapshot v1, https://goo.gl/fbAQLP



exports[`CaseStudyListItem matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="card m-3"
  >
    <div
      class="d-flex flex-column flex-md-row align-items-center"
    >
      <div
        class="col-md-5 p-0"
      >
        <img
          src="www.img1.com"
          width="300px"
        />
      </div>
      <div
        class="col-md-7"
      >
        <h2
          class="primary-font"
        >
          Guide price
        </h2>
        <h1
          class="primary-font"
        >
          Â£100,000
        </h1>
        <h3>
          2 bedrooms 
        </h3>
        <p>
          100 blah blah road
        </p>
      </div>
    </div>
    <p
      class="date--listed col-12 p-3 m-0 w-100"
    >
      Listed on: 3-5-2020
    </p>
  </div>
</DocumentFragment>
`;
